inst/examples/03-formats.Rmd

```diff
@@ -110,7 +111,7 @@ bookdown::gitbook:
       instapaper: no
       vk: no
       whatsapp: no
```

Comment:
See here. I added already.

Reply:
Oww, sorry, True. I need to stop programming for today. =)

Reply:
No worries! 
Thanks for your ideas and suggestions. They are all really welcome !